benchmarks/core/itoa.benchmark.cpp

```
@@ -0,0 +1,56 @@
+//
+/*!
+  \file   itoa.benchmark.cpp
+  \brief  Implementations for itoa nanobench benchmarks in the core module.
+*/
+
+#include "../benchmark_factory.hpp"
+#include "core.hpp"
+
+namespace toy {
+
+// itoa benchmarks
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)